Description: re-enable the dl() function in debian builds
 Despite upstream's claim that it "might not be threadsafe on all platforms".
 It is on ours.
Author: Steve Langasek <vorlon@debian.org>
Bug-Debian: http://bugs.debian.org/297839
Forwarded: no

--- php5.orig/ext/standard/dl.c
+++ php5/ext/standard/dl.c
@@ -74,12 +74,7 @@ PHPAPI PHP_FUNCTION(dl)
 		(strcmp(sapi_module.name, "cli") != 0) &&
 		(strncmp(sapi_module.name, "embed", 5) != 0)
 	) {
-#ifdef ZTS
-		php_error_docref(NULL TSRMLS_CC, E_WARNING, "Not supported in multithreaded Web servers - use extension=%s in your php.ini", filename);
-		RETURN_FALSE;
-#else
 		php_error_docref(NULL TSRMLS_CC, E_DEPRECATED, "dl() is deprecated - use extension=%s in your php.ini", filename);
-#endif
 	}
 
 	php_dl(filename, MODULE_TEMPORARY, return_value, 0 TSRMLS_CC);
